ui/src/app/dashboard/views/DashboardConfigurationForm/DashboardConfigurationSubnetForm/DashboardConfigurationSubnetForm.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect } from "react";
+
+import { Spinner, Strip } from "@canonical/react-components";
+import { useDispatch, useSelector } from "react-redux";
+
+import FormikField from "app/base/components/FormikField";
+import FormikForm from "app/base/components/FormikForm";
+import LegacyLink from "app/base/components/LegacyLink";
+import baseURLs from "app/base/urls";
+import { actions as fabricActions } from "app/store/fabric";
+import fabricSelectors from "app/store/fabric/selectors";
+import { actions as subnetActions } from "app/store/subnet";
+import subnetSelectors from "app/store/subnet/selectors";
+import type { Subnet } from "app/store/subnet/types";
+
+type SubnetDiscoveryValues = {
+  [x: number]: Subnet["active_discovery"];
```

Comment:
@petesfrench I've explicitly typed the form values here, otherwise you get a strict mode error. The type is a bit hard to parse, but it just means an object with numbered keys and boolean values (because subnet.active_discovery is a boolean).
``` typescript
{
  1: true,
  2: false,
}